test/museek/test_data_element.py

```diff
@@ -330,6 +335,17 @@ def test_flagged_mean(self):
                              [15., 16., 12.5]])
         np.testing.assert_array_equal(expect, mean.squeeze)
 
+    def test_kurtosis_when_flags_is_not_none(self):
+        flag_array = np.zeros((3, 3, 3), dtype=bool)
+        flag_array[0, 0, 0] = True
+        flag_array[1, 1, 1] = True
+        flag_array[2, 2, 2] = True
+        flags = FlagList(flags=[FlagElement(array=flag_array)])
+        kurtosis = self.element.kurtosis(flags=flags)
+        print ('kurtosis',kurtosis)
```

Comment:
please remove print

Reply:
axis updated , same as the other metrics